### PR TITLE
🍱 Migrate remaining assets from ampbyexample.com

### PR DESCRIPTION
**Pull request by matthiasrohmer**
This PR migrates the remaining static files from abe.com - it doesn't respect eventually unused files but just brings them over 1:1.

Currently the are still scoped, unscoping will happen with the work on `refactor/statics` when everything is back in place.

### CODE REVIEW

File: examples/static/samples/json/amp-bind-data-new-state.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "currentAnimal": "bird",
+  "bird": {
+    "imageUrl": "/img/bird.jpg",
```
Comment (sebastianbenz) [inline examples/static/samples/json/amp-bind-data-new-state.json]: Shouldn't these URLs be migrated to /static/samples/img?

File: examples/static/samples/json/amp-story-auto-ads-0.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "templateId": "video-template",
+  "data": {
+    "videoSource": "https://ampbyexample.com/video/ad-google-home.mp4",
```
Comment (sebastianbenz) [inline examples/static/samples/json/amp-story-auto-ads-0.json]: Remove ampbyexample.com

**Comment by matthiasrohmer** [comment]
No, good catch. Splitting things up was obviously worth it 😁

File: examples/static/samples/json/nearby_attractions.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "items": [{
+        "attraction": "Hyde Park",
+        "type": "park",
+        "url": "/samples_templates/hotel"
```
Comment (sebastianbenz) [inline examples/static/samples/json/nearby_attractions.json]: these need to be fixed as well

**Comment by matthiasrohmer** [comment]
Thanks for the considerate review! 🙂 Fixed the links in the JSONs - curious if you have any ideas on how to template this.

**Comment by matthiasrohmer** [comment]
> Templating this should be fairly easy, simply pipe the whole JSON dir through a templating engine.

Sure but that would still leave us with manually maintaining paths/URLs to project-internal pages 😉

**Pull request merged**